CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ if (WIN32)
 	link_directories(${EXTERNAL}/curl/lib)
 	link_directories(${EXTERNAL}/MySQL/lib)
 elseif(APPLE)
-	link_directories(/usr/local/lib)
+	link_directories(/usr/local/opt/mysql-client/lib)
```

Comment:
have you checked the curl link is not required?

Reply:
It wasn't linked previously either, the only reason this exists is for mysql.